xpk/xpk.py

```diff
@@ -1450,6 +1459,18 @@ def workload_name_type(value, pat=re.compile(r'[a-z]([-a-z0-9]*[a-z0-9])?')):
     help='The number of slices to run the job on, defaults to 1.',
     required=True,
 )
+cluster_create_optional_arguments.add_argument(
+  '--cluster-type',
+    type=str,
+    default='regional',
+    help='regional or zonal cluster type.'
+)
+cluster_create_optional_arguments.add_argument(
+  '--cluster-machine-type',
+    type=str,
+    default='e2-standard-32',
+    help='cluster machine type.'
```

Comment:
help='Set the machine tpu within the default cpu node pool. For zonal clusters, make sure that the zone supports the machine type, and for regional clusters, the all zones in the region supports the machine type.'